src/modules/jcms_rest/src/Plugin/Field/FieldTypeExtend/TelephoneItem.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\jcms_rest\Plugin\Field\FieldTypeExtend;
+
+use Drupal\telephone\Plugin\Field\FieldType\TelephoneItem as TelephoneItemExtend;
+use Drupal\Core\Field\FieldDefinitionInterface;
+
+/**
+ * {@inheritdoc}
+ */
+class TelephoneItem extends TelephoneItemExtend {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function generateSampleValue(FieldDefinitionInterface $field_definition) {
+    $values = parent::generateSampleValue($field_definition);
+    switch ($field_definition->getName()) {
+      case 'field_block_phone_number':
+          $values['value'] = '+'.$values['value'];
```

Comment:
seems to have a minimum length, if the api-raml reference is correct https://github.com/elifesciences/api-raml/blob/fdc4dc6d2a52db93fb332d412569f932385ba6ea/src/misc/phone-number.v1.yaml#L10

Reply:
The value returned by the parent will be a rand number between:
100000000 and 999999999

All this override is doing is prefixing a `+` to that value.